examples/glasbey-colormap.py

```diff
@@ -0,0 +1,81 @@
+"""
+Glasbey colormap
+================
+
+A long-requested feature [1]_ for napari was to display labels/segmentation
+layers using a well-known discrete colormap such as Glasbey [2]_.
+
+In this example, we demonstrate displaying segmentations using custom colormaps
+with the help of the glasbey Python library [3]_. We display a segmentation
```

Comment:
```suggestion
with the help of the glasbey Python library [3]_ that you can install using conda or pip. We display a segmentation
```

Reply:
I feel like explicitly mentioning that one needs to install this is important, because otherwise to me it suggests it should just work with napari out of the box and yet it doesn't.

Reply:
Maybe leave off conda or pip and end your suggestion after install. I'm pretty much using mostly uv and pixi these days.

Reply:
I was going off of our other examples, e.g.:
https://github.com/napari/napari/blob/68de9b0fad0cc5e7198dfa19565d334245d8e345/examples/surface_timeseries_.py#L31-L33

Reply:
Fair enough for this PR. We probably will need to update all within the next year :wink: